client/src/get-compiled-template.js

```diff
@@ -1,14 +1,20 @@
 'use strict';
 
 var format = require('stringformat');
+var handlebars = require('oc-template-handlebars');
+var jade = require('oc-template-jade');
 var request = require('minimal-request');
 
-var executor = require('./executor');
 var settings = require('./settings');
 var TryGetCached = require('./try-get-cached');
 
 module.exports = function(cache){
 
+  var renderers = {
```

Comment:
Used templateEngines instead of renderers, I think will be better if we try to stick to the same name convention